clone-repositories.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Just a simple shell script to clone all the required repos for running the test suite.